src/main/resources/reference.conf

```diff
@@ -31,6 +31,35 @@ cassandra-journal {
   # Delay between connection retries
   connect-retry-delay = 5s
 
+  # Cassandra driver connection pool settings
+  # Documented at https://datastax.github.io/java-driver/features/pooling/
+  connection-pool {
+
+    # Create new connection threshold local
+    new-connection-threshold-local = 800
+
+    # Create new connection threshold remote
+    new-connection-threshold-remote = 200
+
+    # Connections per host core local
+    connections-per-host-core-local = 2
```

Comment:
I don't think we should change the default (1) for this. Maxing out 1 tcp connection sounds good enough. I tried with 1 and I actually got more stable results than with 2. Do you have any evidence that this a better default?

@chbatey do you have any experience/opinion about what defaults we should use for the connection pool settings?